### PR TITLE
[Compiler playground] bold changed passes

### PR DESCRIPTION
**Pull request by JackYoustra**
## Summary

In the playground, it's hard to see at a glance what compiler passes are involved in introducing changes.
This PR bolds every pass that introduces a change.

## How did you test this change?

Before:
image

After:
image

### CODE REVIEW

**Comment by gsathya** [comment]
This is neat, thanks!

File: compiler/apps/playground/components/Editor/Output.tsx [hunk 1]
```diff
@@ -187,13 +187,30 @@ function Output({ store, compilerOutput }: Props) {
...
+      if (result.kind === "hir" || result.kind === "reactive") {
+        currResult += `function ${result.fnName}\n\n${result.value}`;
+      }
+      if (currResult !== lastResult) {
+        changedPasses.add(passName);
+      }
+      lastResult = currResult;
+    }
+  }
+  console.log([...changedPasses]);
```
Comment (josephsavona) [inline compiler/apps/playground/components/Editor/Output.tsx:204]: Nit: let’s remove the log statement

**Review by josephsavona — COMMENTED** [review]
Nice! Many of us are in transit so just some quick feedback, we’ll try to do a full review soon

**Comment by mofeiZ** [comment]
Yess the new output looks so good!

**Review by mofeiZ — APPROVED** [review]

**Pull request merged**